uni/lib/controller/local_storage/app_shared_preferences.dart

```diff
@@ -31,7 +31,7 @@ class AppSharedPreferences {
   ];
   static const String filteredExamsTypes = 'filtered_exam_types';
   static final List<String> defaultFilteredExamTypes =
-      Exam.types.keys.toList();
+      Exam.types.keys.toList().sublist(0, 4);
```

Comment:
@thePeras  this looks a bit hardcoded. Are we sure the order and size of this list never change?

Reply:
The list is defined in the Exam class and never change.
The order is always the same as the defined, I think 🤔

Reply:
I couldn't do the .remove() here because of a given problem that I don't understand.
Check the solution I came up